gr-blocks/lib/message_debug_impl.cc

```diff
@@ -24,30 +25,55 @@ namespace blocks {
 // old *** HEADING *** line length.
 constexpr unsigned int PRINT_LINE_LENGTH = 36;
 
-message_debug::sptr message_debug::make(bool en_uvec)
+message_debug::sptr message_debug::make(bool en_uvec, spdlog::level::level_enum log_level)
 {
-    return gnuradio::make_block_sptr<message_debug_impl>(en_uvec);
+    return gnuradio::make_block_sptr<message_debug_impl>(en_uvec, log_level);
 }
 
-message_debug_impl::message_debug_impl(bool en_uvec)
+message_debug_impl::message_debug_impl(
+    bool en_uvec, spdlog::level::level_enum log_level = spdlog::level::info)
```

Comment:
Do we need a default in the impl, or just in the .h?

Reply:
Only in the h, indeed, lemme fix that.